Src/remote/ApplicationProcessManager.cpp

```diff
@@ -424,7 +424,7 @@ qint64 ApplicationProcessManager::launchQMLApp(ApplicationDescription *desc, std
     if (appParams.length() > 0)
 		parameters << appParams;
 
-	qmlGenerateRole(desc->id());
+	//qmlGenerateRole(desc->id());
```

Comment:
I understand that librolegen (which implements qmlGenerateRole, with old pub/priv roles) is now a bit useless, but why did it need to generate roles on the fly here ?

Reply:
I really don't know... Ask morphis... His commit messages weren't usually very detailed:

https://github.com/webOS-ports/librolegen/commit/afaab12fc9786a1b1f39998ac9ea1d96364469f7
https://github.com/webOS-ports/luna-appmanager/commit/f1bcfac9f041c213a69eaf758e7b34a87074c262
https://github.com/webOS-ports/luna-appmanager/commit/76bae5aa2e5871a02c6ced0cb502bde843648446

Reply:
I really don't see any usecase for this. Nowadays the roles are shipped with each app, and we start all apps via SAM... "launchQMLApp" is probably useless now.

Reply:
I've looked around a bit, and it looks like we start an app "the old way" when the SAM call has failed, see https://github.com/webOS-ports/luna-appmanager/blob/master/Src/remote/ApplicationProcessManager.cpp#L192

Reply:
But do we actually need a fallback, I'm not so sure.

Reply:
I am also not sure.... SAM should be able to launch everything, shouldn't it?

Reply:
Yes, it should. I'm quite in favor of just dropping the whole fallback thing. I'll try and do that in another PR, yours here is fine as it is.

Reply:
I'll do a bit of cleanup. Consider this WIP for now